test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua

```diff
@@ -317,6 +340,13 @@ runner.Step("Check applying of PT by processing GetVehicleData", SuccessfulProce
 runner.Step("Check applying of PT by processing SubscribeVehicleData", DisallowedRPC,
   { "SubscribeVehicleData", { gps = true }, "VehicleInfo" })
 
+runner.Step("Trigger PTU from HMI", triggerPTU)
```

Comment:
Seeing an error for this step
```
 Trigger_PTU_from_HMI                                                                  [FAIL] (11999 ms)
    HMI notification SDL.OnStatusUpdate: Timeout: HMI notification SDL.OnStatusUpdate: Timeout expired
    HMI call BasicCommunication.PolicyUpdate: Timeout: HMI call BasicCommunication.PolicyUpdate: Timeout expired
```
The ATF log only shows the UPDATE_NEEDED notification for this step
```
===== Trigger_PTU_from_HMI : 
HMI->SDL [02-06-2022 10:05:36,142] {"method":"SDL.OnPolicyUpdate","params":{},"jsonrpc":"2.0"}
SDL->HMI [02-06-2022 10:05:36,143] {"jsonrpc":"2.0","method":"SDL.OnStatusUpdate","params":{"status":"UPDATE_NEEDED"}}
```

Reply:
Looks like we have different results.
The script is passed on my env
`PASSED	test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua`
My environment: `Core: develop(2fa8210), ATF: develop (2645fc1)`.
And what is your environment?

Reply:
Using the same commits for the environment
Core: develop(2fa8210)
ATF: develop(2645fc1)

Here are my logs for the test
[1921_Invalid_PT_after_cutting_unknow_values_20220603155610_full.txt](https://github.com/smartdevicelink/sdl_atf_test_scripts/files/8835390/1921_Invalid_PT_after_cutting_unknow_values_20220603155610_full.txt)
[Console.txt](https://github.com/smartdevicelink/sdl_atf_test_scripts/files/8835391/Console.txt)
[SmartDeviceLinkCore.log](https://github.com/smartdevicelink/sdl_atf_test_scripts/files/8835392/SmartDeviceLinkCore.log)

Reply:
Thanks! 
After a quick look i see only extra message in your log `StartPTExchange: Starting exchange skipped, since applications search is in progress` during failed PTU triggering. The more deeply analyze is required, so I will return to you later.
Is this fail stable on your environment?

Reply:
And also you have some activity with BT that triggers found application search according to core log.
Please try to execute script without BT searching.

Reply:
@GetmanetsIrina turning off bluetooth seems to have fixed the issue. SDL was probably connecting to another application(Test Suite) on my mobile device.